g11n-ws/modules/md-data-api-s3impl/src/main/java/com/vmware/vip/messages/data/conf/RsaCryptUtils.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2019-2020 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package com.vmware.vip.messages.data.conf;
+ 
+import java.io.UnsupportedEncodingException;
+import java.security.InvalidKeyException;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.PublicKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.X509EncodedKeySpec;
+import java.util.Base64;
+
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+/**
+ * 
+ * it use to decode the access key and Secret key
+ *
+ */
+public class RsaCryptUtils {
```

Comment:
Please check this issue reported by SONAR:
https://sonarcloud.io/organizations/singleton/rules?open=java%3AS5542&rule_key=java%3AS5542